R/z_animintHelpers.R

```diff
@@ -751,6 +755,8 @@ getLegend <- function(mb){
 ##' @importFrom stats na.omit
 ##' @import data.table
 getCommonChunk <- function(built, chunk.vars, aes.list){
+  group <- NULL
+  ## Above to avoid CRAN NOTE.
```

Comment:
Hi @tdhock , does the CRAN NOTE mean the output from `R CMD check --as-cran`?

Reply:
right! it complains that group is not defined, so here we give a definition (even if it is not used by the code) just to silence that check NOTE.

Reply:
I see, thanks!